node.md

```diff
@@ -0,0 +1,39 @@
+## Draft Node.js Guidelines
+
+### General
+* Use [Standard JS](https://standardjs.com/) to lint your code. A conistent approach to code layout, spacing and formatting to code makes it easier to switch between projects. By [adding it as a dev dependency](https://standardjs.com/index.html#install) it can be easily used in the terminal, [Webstorm](https://blog.jetbrains.com/webstorm/2017/04/using-javascript-standard-style/), and [Visual Studio](https://marketplace.visualstudio.com/items?itemName=chenxsan.vscode-standardjs).
```

Comment:
I'd say that this is a Javascript rather than Node standard so maybe just reference that and have it in the one place?